src/components/KeyboardShortcutsModal.js

```diff
@@ -161,6 +161,9 @@ export default compose(
     withWindowDimensions,
     withLocalize,
     withOnyx({
-        isShortcutsModalOpen: {key: ONYXKEYS.IS_SHORTCUTS_MODAL_OPEN},
+        isShortcutsModalOpen: {
+            key: ONYXKEYS.IS_SHORTCUTS_MODAL_OPEN,
+            initWithStoredValues: false,
```

Comment:
Why don't we want to init with the store value?

Reply:
This is kind of NAB but I would still like to know why we changed this.

Reply:
Because it was causing problems here: https://github.com/Expensify/App/pull/18578#issuecomment-1538528133

Slack conversation: https://expensify.slack.com/archives/C01GTK53T8Q/p1683563738386549?thread_ts=1683551147.132719&cid=C01GTK53T8Q

Reply:
I see, ok that works for me. It's definitely not necessary for the keyboard shortcuts modal to persist it's open state between refreshes.